test/docsxref/make.jl

```diff
@@ -54,17 +54,17 @@ end
     end
     @test isnothing(captured.value)
     @test contains(
-        replace(captured.output, "src\\index" => "src/index"),
+        replace(captured.output, "\\src\\index" => "src/index"),
```

Comment:
... and this (and its sibling a few lines below) were the other part of the fix for Windows...